<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OrderSummaryWidget</class>
 <widget class="QWidget" name="OrderSummaryWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>146</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:transparent;</string>
  </property>
  <widget class="QLabel" name="SizeLabel">
   <property name="geometry">
    <rect>
     <x>138</x>
     <y>84</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Chalkboard</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:none;
color:rgb(78, 78, 78)</string>
   </property>
   <property name="text">
    <string>Size</string>
   </property>
   <property name="textFormat">
    <enum>Qt::RichText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="FlavorLabel">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>84</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Chalkboard</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:none;
color:rgb(150, 150, 150)</string>
   </property>
   <property name="text">
    <string>Flavor</string>
   </property>
   <property name="textFormat">
    <enum>Qt::RichText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="PackageLabel">
   <property name="geometry">
    <rect>
     <x>522</x>
     <y>84</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Chalkboard</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:none;
color:rgb(78, 78, 78)</string>
   </property>
   <property name="text">
    <string>Package</string>
   </property>
   <property name="textFormat">
    <enum>Qt::RichText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="Crispness">
   <property name="geometry">
    <rect>
     <x>388</x>
     <y>84</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Chalkboard</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:none;
color:rgb(78, 78, 78)</string>
   </property>
   <property name="text">
    <string>Crisp Factor</string>
   </property>
   <property name="textFormat">
    <enum>Qt::RichText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="QuantityLabel">
   <property name="geometry">
    <rect>
     <x>268</x>
     <y>84</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Chalkboard</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:none;
color:rgb(78, 78, 78)</string>
   </property>
   <property name="text">
    <string>Quantity</string>
   </property>
   <property name="textFormat">
    <enum>Qt::RichText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="FlavorImage">
   <property name="geometry">
    <rect>
     <x>26</x>
     <y>32</y>
     <width>70</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="SizeImage">
   <property name="geometry">
    <rect>
     <x>158</x>
     <y>32</y>
     <width>70</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="QuantityImage">
   <property name="geometry">
    <rect>
     <x>288</x>
     <y>32</y>
     <width>70</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="CrispImage">
   <property name="geometry">
    <rect>
     <x>418</x>
     <y>32</y>
     <width>70</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="PackageImage">
   <property name="geometry">
    <rect>
     <x>548</x>
     <y>32</y>
     <width>70</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="FlavorMask">
   <property name="geometry">
    <rect>
     <x>26</x>
     <y>34</y>
     <width>70</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgba(255,255,255,150)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="SizeMask">
   <property name="geometry">
    <rect>
     <x>158</x>
     <y>34</y>
     <width>70</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgba(255,255,255,150)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="QuantityMask">
   <property name="geometry">
    <rect>
     <x>288</x>
     <y>34</y>
     <width>70</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgba(255,255,255,150)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="CrispMask">
   <property name="geometry">
    <rect>
     <x>418</x>
     <y>34</y>
     <width>70</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgba(255,255,255,150)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="PackageMask">
   <property name="geometry">
    <rect>
     <x>548</x>
     <y>34</y>
     <width>70</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgba(255,255,255,150)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="FooterLogo">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>112</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image:url(:/Common/Images/FooterLogo.png)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="HelpLabel">
   <property name="geometry">
    <rect>
     <x>588</x>
     <y>112</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image:url(:/Start/Images/Help.png)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="OrderSummaryLabel">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>3</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Chalkboard</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Order Summary</string>
   </property>
   <property name="textFormat">
    <enum>Qt::RichText</enum>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
